hydroshare/local_settings.py

```diff
@@ -179,3 +179,5 @@
 #DEFAULT_SUPPORT_EMAIL=''
 
 HYDROSHARE_SHARED_TEMP = '/shared_tmp'
+
+TIME_ZONE = "Etc/UTC"
```

Comment:
UTC is what's being used right now, at least on dev. My understanding is that UTC is the best to use server-size, but I defer to @mjstealey